zio-http/shared/src/main/scala/zio/http/ClientSSLConfig.scala

```diff
@@ -22,31 +22,61 @@ import zio.Config.Secret
 sealed trait ClientSSLConfig
 
 object ClientSSLConfig {
+  val clientSSLCertConfig: Config[ClientSSLCertConfig] = {
```

Comment:
It's already namespaced inside `ClientSSLConfig`, so it can have a shorter name: `config`.

Reply:
Done!